docker/templates/Dockerfile.json-dump.m4

```diff
@@ -13,6 +13,12 @@ COPY docker/musicbrainz-json-dump/crontab /var/spool/cron/crontabs/musicbrainz
 RUN chown musicbrainz:musicbrainz /var/spool/cron/crontabs/musicbrainz && \
     chmod 600 /var/spool/cron/crontabs/musicbrainz
 
+ENV MB_CONTAINER_TYPE json-dump
```

Comment:
the script appears to use `MUSICBRAINZ_SERVICE` now (not sure which you prefer)

Reply:
Not sure why I typed a different name in the script. Now fixed.